ui/appui-react/src/appui-react/stagepanels/StagePanelLocation.ts

```diff
@@ -11,7 +11,7 @@
  */
 export enum StagePanelLocation {
   Top = 101,
-  Left,
-  Right,
-  Bottom,
+  Left = 103,
```

Comment:
Not sure if this is the right way to handle this, since it's probably a breaking change, but TypeScript is now correctly pointing out that the definition for `StagePanelLocation` in appui-react is out of sync with [the definition in appui-abstract](https://github.com/iTwin/itwinjs-core/blob/master/ui/appui-abstract/src/appui-abstract/widget/StagePanel.ts#L23)!

Was originally flagged by TypeScript here: https://github.com/iTwin/appui/blob/20eba19c8092955e3e86d9afd4f282e571e6c308/ui/appui-react/src/test/ui-items-provider/UiItemsManager.test.tsx#L776-L780

Reply:
**Ouch!** That's a good catch by typescript! I think we can simply go with this for the major release (would only break if someone were to use numeric values directly)

Reply:
After a second thought:
I think we should revert this change since there's no need to keep the two enums in sync. We are mapping enum values correctly: https://github.com/iTwin/appui/blob/0eeaa10787b0612d6364d5427cd2afef835b88cc/ui/appui-react/src/appui-react/ui-items-provider/AbstractUiItemsManager.ts#L661-L697

Reply:
In the test file, simply use enum from `@itwin/appui-abstract` with `AbstractUiItemsManager`

Reply:
Made that change in https://github.com/iTwin/appui/pull/1064/commits/36eeed35f08a38bc086e4285dcd086fb670b3c04